h/views/__init__.py

```diff
@@ -19,7 +19,8 @@ def includeme(config):
 
     # help
     config.add_route('help', '/docs/help')
-    config.add_route('onboarding', '/welcome')
+    config.add_route('onboarding', '/welcome/')
+    config.add_route('custom_onboarding', '/welcome/{welcome_slug}')
```

Comment:
It's a small thing, but let's just call the matchdict item `slug`. It's the `slug` in the welcome page URL so there's no need to reiterate that.